skills/csharp/experimental/bingsearchskill/Services/BotServices.cs

```diff
@@ -48,20 +48,20 @@ public BotServices(BotSettings settings, IBotTelemetryClient client)
                     }
                 }
 
-                if (config.Knowledgebases != null)
-                {
-                    foreach (var kb in config.Knowledgebases)
-                    {
-                        var qnaEndpoint = new QnAMakerEndpoint()
-                        {
-                            KnowledgeBaseId = kb.KbId,
-                            EndpointKey = kb.EndpointKey,
-                            Host = kb.Hostname,
-                        };
-                        var qnaMaker = new QnAMaker(qnaEndpoint);
-                        set.QnAServices.Add(kb.Id, qnaMaker);
-                    }
-                }
+                //if (config.Knowledgebases != null)
```

Comment:
the latest fix should be

foreach (var kb in config.Knowledgebases)
                {
                    var qnaEndpoint = new QnAMakerEndpoint()
                    {
                        KnowledgeBaseId = kb.KbId,
                        EndpointKey = kb.EndpointKey,
                        Host = kb.Hostname,
                    };

                    set.QnAConfiguration.Add(kb.Id, qnaEndpoint);
                }

Reply:
i think i mistakenly commented out this part in my email and todo skill PR. can you check all the skills to make sure this is handled properly?

Reply:
haha, sad story..

Reply:
sure. this skill has been fixed.